fix(server): use HTTP status OK instead of CREATED for assets

uhthomas opened the pull request:
The NGINX gzip module does not compress responses with a status of 201, which is
a major issue specifically for the /api/asset/time-bucket endpoint where
responses can be upwards of 5Mi. The size of the response is dramatically
reduced with gzip to 500Ki in some cases.

https://trac.nginx.org/nginx/ticket/471
https://trac.nginx.org/nginx/ticket/394

The signature of these endpoints should be GET rather than POST anyway, but that
is a bigger discussion.

uhthomas commented on server/src/immich/api-v1/asset/asset.controller.ts:
```diff
@@ -71,7 +72,7 @@ interface UploadFiles {
 @Authenticated()
 export class AssetController {
-  constructor(private assetService: AssetService) {}
+  constructor(private assetService: AssetService) { }
```
idk?

jrasm91 replied:
That is a linting error.

uhthomas committed:
fix(server): use HTTP status OK instead of CREATED for assets

The NGINX gzip module does not compress responses with a status of 201, which is
a major issue specifically for the /api/asset/time-bucket endpoint where
responses can be upwards of 5Mi. The size of the response is dramatically
reduced with gzip to 500Ki in some cases.

https://trac.nginx.org/nginx/ticket/471
https://trac.nginx.org/nginx/ticket/394

The signature of these endpoints should be GET rather than POST anyway, but that
is a bigger discussion.
